### DISCUSSION TITLE
How to add resulting text to pdf?

### DISCUSSION THREAD

**Discussion by piperino11** [mindee/doctr#1192] [Q&A] [answered]
Hi , is there a way to add the text obtained from docTr to the pdf and save it?

**Comment by felixdittrich92** [comment]
Hi @piperino11 👋 ,

You mean saving as PDF/A file (which has a searchable text layer on top) ?

**Reply by piperino11** [reply]
Yes I have an original pdf, after OCR I would like to add the result to the Pdf as a text layer

**Reply by felixdittrich92** [reply, marked as answer]
You have two options:
1. Following our tutorial (which is a bit outdated) https://colab.research.google.com/github/mindee/notebooks/blob/main/doctr/export_as_pdfa.ipynb 
2. Or follow this blog post https://mindee.com/blog/create-ocrized-pdfs-in-2-steps/

Let me know if this helps 👍

**Reply by piperino11** [reply]
thanks it is very helpful <3
